.github/scripts/stale_pr_date_setter.py

```diff
@@ -0,0 +1,272 @@
+"""
```

Comment:
Feel free to use the `tools` directory. We collect here some scripts from time to time.